lib/privileged-linux.c

```diff
@@ -0,0 +1,219 @@
+/*
+ * Copyright (c) 2002-2015 BalaBit IT Ltd, Budapest, Hungary
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ * As an additional exemption you are allowed to compile & link against the
+ * OpenSSL libraries as published by the OpenSSL project. See the file
+ * COPYING for details.
+ *
+ */
+
+#include "privileged-linux.h"
+#include "gprocess.h"
+#include "messages.h"
+#include "file-perms.h"
+#include "affile/affile-common.h"
+
```

Comment:
What are the dependencies here? My perception is that this is a leaf module not depending on anything but the kernel. I don't get these includes.